utilities/_layout.scss

```diff
@@ -0,0 +1,32 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
+.grid {
+    @include grid(repeat(6, 1fr), 16px, 24px);
+    @media ($mobile-upper <= width <= $desktop-lower) {
+        @include grid(repeat(12, 1fr), 32px, 75px);
+    }
+    @media (width >= ($desktop-lower + 1)) {
+        @include grid(repeat(12, 1fr), 32px, 100px);
+    }
+}
+
+.full-width {
+    grid-column: span 6;
+    @media (width >= $mobile-upper) {
```

Comment:
@beccadawson 
Could we use the media queries from the Breakpoints mixin?
https://github.com/AgePartnership/oleg-starter-pack/blob/develop/base/_breakpoints.scss

e.g.
`@include breakpoint(desktop-up) { ... }`